docs/source/llms/openai/notebooks/openai-code-helper.ipynb

```diff
@@ -0,0 +1,865 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Building a Code Assistant with OpenAI & MLflow\n",
```

Comment:
Can we add a `Budget` section to give user an estimated cost of executing this notebook? GPT4 is pretty expensive, as a user i will be interested in that.

Reply:
added